JAICore/jaicore-ml/src/main/java/jaicore/ml/core/evaluation/measure/ClassifierMetricGetter.java

```diff
@@ -0,0 +1,238 @@
+package jaicore.ml.core.evaluation.measure;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.NoSuchElementException;
+
+import meka.core.Metrics;
+import meka.core.Result;
+import weka.classifiers.Evaluation;
+
+/**
+* Class for getting metrics by their name for single- and multilabel
+* classifiers.
+*
+* @author helegraf, mwever
+*
+*/
+public class ClassifierMetricGetter {

```

Comment:
This class looks a bit weird to me. For simple multi-class, it is just a forwarder for an Evaluation object. Perhaps, the benefits of using this class could be explained in the class JavaDoc.